src/Harpoon.cpp

```diff
@@ -49,6 +49,16 @@ void Harpoon::RequestChunks() {
    }
 }
 
+
+Harpoon::Battling Harpoon::Cancel() {
```

Comment:
Ha ha. That's fine. Alex and I actually discussed it at length.

My first choice was "abort" but he claimed it was too politically charged ;)